projects.yaml

```diff
@@ -1528,3 +1528,9 @@ projects:
   github_id: unverbuggt/mkdocs-familytree-example
   labels: []
   category: other
+- name: extract-listings
+  mkdocs_plugin: placeholder
```

Comment:
Shouldn't it be `extract-listings` or something?

Reply:
Good catch, was a copy paste error